<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QConfigurationScreenBase</class>
 <widget class="QWidget" name="udg::QConfigurationScreenBase">
  <property name="windowTitle">
   <string>PACS Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../main/main.qrc">
    <normaloff>:/images/icons/preferences-system-network.svg</normaloff>:/images/icons/preferences-system-network.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabConfiguration">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>PACS Servers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="4" column="1">
        <widget class="QPushButton" name="m_buttonDeletePacs">
         <property name="toolTip">
          <string>Delete selected PACS</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
         <property name="icon">
          <iconset resource="../main/main.qrc">
           <normaloff>:/images/icons/edit-delete.svg</normaloff>:/images/icons/edit-delete.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="m_buttonAddPacs">
         <property name="toolTip">
          <string>Add the new PACS to the list</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="../main/main.qrc">
           <normaloff>:/images/icons/list-add.svg</normaloff>:/images/icons/list-add.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="m_buttonTestPacs">
         <property name="toolTip">
          <string>Test if selected PACS is alive</string>
         </property>
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTreeWidget" name="m_PacsTreeView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="indentation">
          <number>20</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>PacsId</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>AE Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Institution</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Q/R Port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Send Port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Default Query PACS</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0" rowspan="4">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="textLabel6_7">
           <property name="text">
            <string>Institution:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="textLabel4_3">
           <property name="text">
            <string>Send Service:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="m_checkBoxStoreEnabled">
           <property name="text">
            <string>Enabled</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="m_textStoreServicePort">
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="m_checkBoxQueryRetrieveEnabled">
           <property name="text">
            <string>Enabled</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="textLabel8_6">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="m_textQueryRetrieveServicePort">
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel1_2">
           <property name="text">
            <string>AE Title:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLineEdit" name="m_textDescription"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel3_5">
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLineEdit" name="m_textLocation"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel4_2">
           <property name="text">
            <string>Q/R Service:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLineEdit" name="m_textInstitution"/>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="textLabel7_6">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="m_textAddress"/>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="m_textAETitle">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QCheckBox" name="m_checkDefault">
           <property name="text">
            <string>Default Query PACS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Modify Selected PACS Server/Add New PACS Server</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="m_buttonUpdatePacs">
         <property name="toolTip">
          <string>Update parameters of the selected PACS</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
         <property name="icon">
          <iconset resource="../main/main.qrc">
           <normaloff>:/images/icons/document-save.svg</normaloff>:/images/icons/document-save.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage_2">
      <attribute name="title">
       <string>Local Configuration</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Local DICOM Configuration</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="m_textInstitutionPhoneNumber">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Country:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="m_textInstitutionCountry">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Phone Number:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="m_textInstitutionZipCode">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="m_textInstitutionAddress">
           <property name="maxLength">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Institution:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="m_textInstitutionEmail">
           <property name="maxLength">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="m_textInstitutionName">
           <property name="maxLength">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="m_textInstitutionTown">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>ZIP Code:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>City:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Institution Contact Information</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="textLabel7">
           <property name="text">
            <string>Max. Simultaneous Connections:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="m_textLocalPort">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>Local Machine AE Title:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel5">
           <property name="text">
            <string>Max. Seconds for Timeout:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_textTimeout">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel6">
           <property name="text">
            <string>Incoming Connections Port:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_textAETitleMachine">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="m_textMaxConnections">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2" rowspan="2">
          <widget class="QFrame" name="m_warningFrameIncomingConnectionsPortInUse">
           <layout class="QGridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_16">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/images/icons/emblem-warning.svg)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>This port is in use by another application.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_PacsTreeView</tabstop>
  <tabstop>m_textAETitle</tabstop>
  <tabstop>m_textAddress</tabstop>
  <tabstop>m_checkBoxQueryRetrieveEnabled</tabstop>
  <tabstop>m_checkBoxStoreEnabled</tabstop>
  <tabstop>m_textTimeout</tabstop>
  <tabstop>m_textInstitutionName</tabstop>
  <tabstop>m_textInstitutionAddress</tabstop>
  <tabstop>m_textInstitutionTown</tabstop>
  <tabstop>m_textInstitutionZipCode</tabstop>
  <tabstop>m_textInstitutionCountry</tabstop>
  <tabstop>m_textInstitutionEmail</tabstop>
  <tabstop>m_textInstitutionPhoneNumber</tabstop>
 </tabstops>
 <resources>
  <include location="../main/main.qrc"/>
  <include location="../main/main.qrc"/>
 </resources>
 <connections/>
</ui>
